docs/azure/sdk/media/mermaidjs/local-developer-authentication.md

```diff
@@ -0,0 +1,49 @@
+---
+ms.topic: include
+ms.date: 08/07/2024
+---
+
+```mermaid
+%% STEPS TO GENERATE IMAGE 
+%% ======================= 
+%% 1. Install mermaid CLI v10.9.1 (see https://github.com/mermaid-js/mermaid-cli/blob/master/README.md): 
+%%    npm i -g @mermaid-js/mermaid-cli@10.9.1 
+%% 2. Run command: mmdc -i local-developer-authentication.md -o ../../media/mermaidjs/local-developer-authentication.svg 
+
+flowchart TD
+    ARL[Local .NET app]
+    
+    VS[Visual Studio]
+    VSC[Visual Studio Code]
+    AZCLI[Azure CLI]
+    AZPS[Azure PowerShell]
+    AZD[Azure Developer CLI]
+
+    DevAccount["Developer account credentials"]
+
+    AS["Azure services"]
+    
+    ARL --> VS
+    ARL --> VSC
+    ARL --> AZD
+    ARL --> AZCLI
+    ARL --> AZPS
+    
+    VS --> DevAccount
+    VSC --> DevAccount
+    AZD --> DevAccount
+    AZCLI --> DevAccount
```

Comment:
The CLI and perhaps others can authenticate service principles in addition to developer accounts. I'm not sure if this diagram adds any new information other than these tools can interact with developer credentials.